pyrasterframes/src/main/python/docs/masking.pymd

```diff
@@ -14,8 +14,10 @@ spark = create_rf_spark_session()
 ```
 
 Masking is a common operation in raster processing. It is setting certain cells to the @ref:[NoData value](nodata-handling.md). This is usually done to remove low-quality observations from the raster processing. Another related use case is to @ref:["clip"](masking.md#clipping) a raster to a given polygon.  
+
+In this section we will demonstrate two common schemes for masking. In Sentinel 2, there is a separate classification raster that defines low quality areas. In Landsat 8, several quality factors are measured and the indications are packed into a single integer, which we have to unpack.
  
-## Masking Example
+## Masking Sentinel 2
 
 Let's demonstrate masking with a pair of bands of Sentinel-2 data. The measurement bands we will use, blue and green, have no defined NoData. They share quality information from a separate file called the scene classification (SCL), which delineates areas of missing data and probable clouds. For more information on this, see the [Sentinel-2 algorithm overview](https://earth.esa.int/web/sentinel/technical-guides/sentinel-2-msi/level-2a/algorithm). Figure 3 tells us how to interpret the scene classification. For this example, we will exclude NoData, defective pixels, probable clouds, and cirrus clouds: values 0, 1, 8, 9, and 10.
 
```

Comment:
Add a `@@@Note` block here indicating GDAL is required for this example. See the `getting-started.pymd` for the syntax if needed.

Reply:
So the files are actually GeoTiffs (COGs I think) on our test bucket.

Reply:
Aha.